modules/op-release-notes-1-7.adoc

```diff
@@ -339,4 +339,24 @@ time="2021-11-04T13:05:26Z" level=error msg="exit status 127"
 // https://github.com/tektoncd/operator/pull/599
 
 
+[id="release-notes-1-7-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.7.1
+
+[id="fixed-issues-1-7-1_{context}"]
+=== Fixed issues
+
+* Before this update, upgrading the {pipelines-title} Operator deleted the data in the database associated with {tekton-hub}, and installed a new database. With this update, an Operator upgrade preserves the data.
+// https://issues.redhat.com/browse/SRVKP-2280
+
+* Before this update, only cluster administrators could access pipeline metrics in the OpenShift Container Platform console. With this update, users with other cluster roles also can access the pipeline metrics.
+// https://issues.redhat.com/browse/SRVKP-2129
+
+* Before this update, pipeline runs failed for pipelines containing tasks that emit large termination messages. The pipeline runs failed because the total size of termination messages of all containers in a pod cannot exceed 12 KB. With this update, the `place-tools` and `step-init` initialization containers that uses the same image are merged to reduce the number of containers running in each tasks's pod. The solution reduces the chance of failed pipeline runs by minimizing the number of containers running in a task's pod. However, it does not remove the limitation of the maximum allowed size of a termination message.
+// https://issues.redhat.com/browse/SRVKP-2243
+
+* Before this update, attempts to access resource URLs directly from the {tekton-hub} web console resulted in an Nginx `404` error. With this update, the {tekton-hub} web console image is fixed to allow accessing resource URLs directly from the {tekton-hub} web console.
+// https://issues.redhat.com/browse/SRVKP-2196
+
+* Before this update, the resource pruner job was designed in such a way that for each namespace, the {pipelines-title} Operator created command that ran in a separate container. With this update, all namespace based configuration are passed to the pruner job together, which results in execution of all the commands in a loop within one container.
```

Comment:
Should we make it a little bit more simpler.

`
Before this update, the resource pruner job created n containers for n namespaces to prune resources, now the prune commands runs as loop in one container.
`

Reply:
I like what @pradeepitm12 suggested, but I don't know what the `n` stands for...number? To be consistent with the phrasing of the other notes, I would restructure Pradeep wrote, something like this:

`Before this update, the resource pruner job created containers for namespaces to prune resources. With this update, the prune commands run as a loop in one container.`